.circleci/config.yml

```diff
@@ -1432,7 +1432,7 @@ workflows:
           filters:
             branches:
               only:
-              - nightly
+              - '*'
```

Comment:
Do you think the doc job  is not being run because we have `'` around the `*`?

Reply:
The filter with  the `'*'` is the same as the base job build_docs runs off (`binary_linux_wheel_py3.7_cpu`) which works fine.
```
      - binary_linux_wheel:
          conda_docker_image: pytorch/conda-builder:cpu
          cu_version: cpu
          filters:
            branches:
              only: '*'
            tags:
              only: /v[0-9]+(\.[0-9]+)*-rc[0-9]+/
          name: binary_linux_wheel_py3.7_cpu
          python_version: '3.7'
          wheel_docker_image: pytorch/manylinux-cuda102

```